File: js/OOP/object_with_array.js
function print(info) {
    console.log(info)
}

class Todo {
    constructor(name, isDone=false) {
        this.name = name
        this.isDone = isDone
    }

    markDone() {
        this.isDone = true
    }
}

class Todos {
    constructor() {
        this.tasks = []; // { name: 'what ever', isDone: true }
    }
}

const todoList1 = new Todos();

// todoList1.tasks.push({name: 'what ever',isDone: false})
const todo1 = new Todo('what ever', false);
todoList1.tasks.push(todo1)

print(todoList1.tasks)

File: python/OOP/object_with_array.py

class Todo:
    def __init__(self, name, isDone=False):
        self.name = name
        self.isDone = isDone

    def markDone(self):
        self.isDone = True
      


class Todos:
    def __init__(self):
        self.tasks = []



todoList1 = Todos()

# todoList1.tasks.append({'name': 'what ever','isDone': False})
todo1 = Todo('nameTask', False)
todoList1.tasks.append(todo1)
print(todoList1.tasks)
